docker/dockerfile.ctr

```diff
@@ -46,8 +46,6 @@ RUN apt update -y --fix-missing && \
             libaio-dev libtbb-dev \
             clang-format \
        software-properties-common  && \
-    apt remove --purge cmake -y && \
```

Comment:
I am fine with bringing back the standard `cmake`, but mind that we had a custom version, because some builds required a newer `cmake` version. Is that resolved now?

Reply:
Is this still an issue?

Reply:
Currently we have no cmake at all. So I suppose that is the reason why Emma removed it.

However, I wonder. We had a specific cmake version before because some project required a newer cmake than the standard ubuntu repos offer.

@EmmaQiaoCh Is that solved?

Reply:
Hi sorry, I missed this...
I'm not sure which project required a newer cmake, but our build pipelines were passed when used the cmake-3.24.1 which installed in merlin-base. So I think it's fine.